src/main/java/org/folio/rest/impl/StorageHelper.java

```diff
@@ -61,4 +71,16 @@ protected static PostgresClient postgresClient(Context vertxContext, Map<String,
     return PostgresClient.getInstance(vertxContext.owner(), TenantTool.tenantId(okapiHeaders));
   }
 
+  protected static <T> void postSync(String table, List<T> entities, Map<String, String> okapiHeaders,
+      Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext, Supplier<Response> respond201) {
+    PostgresClient postgresClient = PgUtil.postgresClient(vertxContext, okapiHeaders);
+    postgresClient.saveBatch(table, entities, result -> {
+      if (result.failed()) {
+        logger.error(result.cause().getMessage(), result.cause());
```

Comment:
```suggestion
        logger.error("Error has occurred during executing post-sync", result.cause());
```

Sometimes exceptions has no message (like NPE), I think it more informatively to add a message.

Reply:
`result.cause().getMessage()` may contain useful information like `"duplicate key value violates unique contraint "instance_pkey"`. Therefore I do not drop that information but prepend "postSync: " to give more context.

Reply:
Thanks @julianladisch, works for me.